### PR TITLE
Correct usage example

### PR DESCRIPTION
**Pull request by jingsam**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -24,22 +24,31 @@ vue-mdl tries to use the latest versions of material design lite and vue
...
 vmdl.registerAll(Vue);
+```
+Or
+```javascript
+var vmdl = require('vue-mdl');
 
 // Access any component or directive
-var checkbox = vmdl.components['mdl-checkbox'];
-var badge = vmdl.directives['mdl-badge'];
+var checkbox = vmdl.components['mdlCheckbox'];
```
Comment (posva) [inline readme.md]: we can actually use `vmdl.components.mdlCheckbox` now :smile:
The same goes for mdlBadge

**Comment by posva** [comment]
Thanks for the PR, I forgot about updating this. Tests are passing, somehow e2e tests are not launching... I may drop them in the future

**Comment by posva** [comment]
Hey @jingsam, after solving some issues I updated how to access components and this is no longer necessary. I'm sorry I couldn't merge your PR but I really appreciate you submitted one :smile:

**Comment by jingsam** [comment]
That's OK

**Pull request closed without merging**